### PR TITLE
Adding Spanish version of the JuMP-dev III page

### PR DESCRIPTION
**Pull request by rocarvaj**
I also added a link from the English version to this page.

### CODE REVIEW

File: meetings/santiago2019/index-SPA.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+El propósito de este workshop es juntar a estudiantes, investigadores y 
+profesionales, con interés en los aspectos de software de [JuMP](https://github.com/JuliaOpt/JuMP.jl) y sus paquetes relacionados. 
+El Workshop está abierto a todo público. En particular, invitamos a nuevos 
+contribuidores y a aquellos que no han tenido la posibilidad de conocer al equipo de desarrollo principal de JuMP.
+
+El Workshop incluirá tutoriales *hands-on* (desde algunos sin experiencia requerida,
+hasta otros acerca de uso avanzado de JuMP), charlas por participantes y desarrolladores principales, y amplio tiempo para colaboración.
+
+Los participantes son invitados a enviar propuestas para charlas cortas (~20 minutos) acerca de los siguientes tópicos:
```
[the remark this answers is not in the document]
Comment (rocarvaj) [reply meetings/santiago2019/index-SPA.md]: Done! At least in the Spanish version.

**Comment by rocarvaj** [comment]
I've followed @mlubin idea and now the Spanish version is at the bottom of the original page.

**Comment by mlubin** [comment]
Gracias!

**Comment by rocarvaj** [comment]
De nada! That was my first pull request ever :sob: I had been a github hermit until now.

**Pull request merged**